x-pack/plugins/security_solution/public/management/pages/policy/view/host_isolation_exceptions/components/assign_flyout.tsx

```diff
@@ -45,6 +46,13 @@ export const PolicyHostIsolationExceptionsAssignFlyout = ({
   const http = useHttp();
   const toasts = useToasts();
   const queryClient = useQueryClient();
+  const privileges = useUserPrivileges().endpointPrivileges;
+
+  useEffect(() => {
+    if (!privileges.canIsolateHost) {
```

Comment:
QQ: Wondering if we should also check the license since the artifacts by policy is a platinum plus feature